src/core/test.js

```diff
@@ -95,6 +95,31 @@ describe( "wb.core helpers test suite", function() {
 
 	} );
 
+	/*
+	 * Test external links after DOMPurify sanitization
+	 */
+	describe( "External links", function() {
+		before( function() {
+			$( "body" ).append( `<div class='external-links-test'>
+				<a id='externalLinkTest1' href='http://www.canada.ca' target='_blank' rel='noreferrer'>Valid link 1</a>
+				<a id='externalLinkTest2' href='http://www.canada.ca' target='_blank' rel='noreferrer noopener'>Valid link 2</a>
+				<a id='externalLinkTest3' href='http://www.canada.ca' target='_blank'>Invalid link 1</a>
+			</div>` );
```

Comment:
Additional unit test case where the result should be undefined

```suggestion
				<a id='externalLinkTest3' href='http://www.canada.ca' target='_blank'>Invalid link 1</a>
				<a id='externalLinkTest4' href='http://www.canada.ca' target='_blank' rel="hellonoreferrerworld>Invalid link 2</a>
				<!-- This one should be valid according to spec because the token are case insensitive but this is a limitation of relList DOM anchor function implemented in browser -->
				<!-- No need to test, but let's keep it here as a documentation item for future reference -->
				<!-- <a id='externalLinkTest5' href='http://www.canada.ca' target='_blank' rel='external noReferrer nofollow'>Valid link 3</a>-->
			</div>` );
```